packages/usa-icon/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@uswds/usa-icon",
+  "version": "0.0.1-alpha.0",
+  "description": "USWDS icon library",
+  "main": "_index.scss",
+  "scripts": {
+    "pretest": "gulp buildIcons",
+    "test:js": "mocha './src/test/*.spec.js'",
+    "test": "npm run test:js"
+  },
```

Comment:
As of now, there's currently no way to generate the sprite from this package directory right?